.coveragerc

```diff
@@ -142,6 +142,10 @@ omit =
     homeassistant/components/braviatv/coordinator.py
     homeassistant/components/braviatv/media_player.py
     homeassistant/components/braviatv/remote.py
+    homeassistant/components/bring/__init__.py
```

Comment:
What is missing to get this fully tested?

Reply:
Everything apart the config_flow.

Reply:
What would you prefer, adding unit tests now or in a future PR? I would prefer to isolate the unit tests from the initial PR of the integration. But thats just my 5 cents :)

Reply:
Sorry, I just meant the `__init__.py`, I think loading the entry should be already covered, but a test for unloading would be nice in my opinion.

Reply:
ahh, ok. That should not take too long :)

Reply:
I based my implementation on ourgroceries and copied quite a bit from there.

Reply:
I am planning to do the rest of the files as well, but not as part of this initial PR